client/ayon_maya/api/lib.py

```diff
@@ -3249,8 +3273,8 @@ def update_content_on_context_change():
     """
 
     host = registered_host()
-    create_context = CreateContext(host)
-    task_entity = get_current_task_entity(fields={"attrib"})
+    create_context = CreateContext(host, discover_publish_plugins=False)
+    task_entity = create_context.get_current_task_entity()
```

Comment:
there should be `task_entity["name"]` named in `instance_values` for `"task"`.

Reply:
Sorry, can you elaborate? How would it help in this scenario?

In this case we want to enforce all instances to match the current context.

Reply:
I could use `create_context.get_current_task_name()` instead?

Reply:
Actually, no - I couldn't - I'm using the task entity attributes further down.